packages/graphback-cli/package.json

```diff
@@ -35,7 +35,7 @@
     "lint-staged": "10.2.9",
     "ncp": "2.0.0",
     "rimraf": "3.0.2",
-    "ts-jest": "25.5.1",
+    "ts-jest": "26.0.1",
```

Comment:
Looks like `26.0.1` is not there, should be `26.1.0`. 

I had to manually pick the version locally after yarn proposed it to me. 

Also CI is failing for similar reason 
```
Error: Couldn't find any versions for "ts-jest" that matches "26.0.1"
 
    at MessageError.ExtendableBuiltin (/opt/yarn-v1.22.4/lib/cli.js:721:66)
 
    at new MessageError (/opt/yarn-v1.22.4/lib/cli.js:750:123)
 
    at Function.<anonymous> (/opt/yarn-v1.22.4/lib/cli.js:50545:13)
 
    at Generator.next (<anonymous>)
 
    at step (/opt/yarn-v1.22.4/lib/cli.js:310:30)
 
    at /opt/yarn-v1.22.4/lib/cli.js:321:13
 
    at runMicrotasks (<anonymous>)
 
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
 
``